graphql-dgs-spring-webmvc/src/main/kotlin/com/netflix/graphql/dgs/mvc/conf/WebMvc.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.graphql.dgs.mvc.conf
+
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor
+import org.springframework.web.servlet.config.annotation.AsyncSupportConfigurer
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer
+
+@Configuration
+open class WebMvc : WebMvcConfigurer {
```

Comment:
Not convinced we should do this in the framework. This will also be problematic to the folks that want to explicitly set a different configuration. All in all, I don't think is a good idea for us to have it here.


Side note, the name of the configuration is not very expressive on what it is doing. T